fchost/fchost.go

```diff
@@ -57,6 +63,25 @@ func (fh *FilecoinHost) Bootstrap() error {
 	return nil
 }
 
+func (fc *FilecoinHost) Ping(ctx context.Context, pid peer.ID) bool {
```

Comment:
Used so the Metadata Index of Miners can discover if a miner responds to the Ping protocol, to decide if it's online or not.